libs/designsystem/src/lib/components/list/list.component.spec.ts

```diff
@@ -141,32 +141,60 @@ describe('ListComponent', () => {
   });
 
   describe('function: ngOnInit', () => {
-    it('should enable load more, if there is a subscriber to the loadMore event emitter', () => {
-      component.loadOnDemand.subscribe((_: LoadOnDemandEvent) => {});
+    describe('there is a subscriber to the loadMore event', () => {
+      beforeEach(() => {
+        component.loadOnDemand.subscribe((_: LoadOnDemandEvent) => {});
+      });
 
-      component.ngOnInit();
+      it('should enable load more, if isLoadOnDemandEnabled is not set', () => {
+        component.isLoadOnDemandEnabled = undefined;
 
-      expect(component.isLoadOnDemandEnabled).toBeTrue();
-    });
+        component.ngOnInit();
```

Comment:
Be a good boy scout 🤞
```suggestion
        component.ngAfterViewInit();
```

Reply:
I'm not sure I understand this suggestion - the function being tested is ngOnInit 😅

Reply:
True, that is the function being called in the test, but it's actually being implemented in the `ngAfterViewInit()`function (which automatically gets called after calling `ngOnInit()`)

Reply:
My bad 🙈